CHANGELOG.md

```diff
@@ -361,6 +363,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added support for `devices` flag to Trainer ([#8440](https://github.com/PyTorchLightning/pytorch-lightning/pull/8440))
 - Added private `prevent_trainer_and_dataloaders_deepcopy` context manager on the `LightningModule` ([#8472](https://github.com/PyTorchLightning/pytorch-lightning/pull/8472))
 - Added support for providing callables to the Lightning CLI instead of types ([#8400](https://github.com/PyTorchLightning/pytorch-lightning/pull/8400))
+- Added corresponded tests for the CometLogger, to check the metrics in place modification ([#9150](https://github.com/PyTorchLightning/pytorch-lightning/pull/9150))
+
```

Comment:
```suggestion
```
we don't list changes to the tests, but thanks for your attention :)

Reply:
Oh I see! My bad! Thank you :)